docs/pages/blog/nextgen-pickers.md

```diff
@@ -0,0 +1,130 @@
+  <source src="/static/blog/nextgen-pickers/keyboard-nav.mp4" type="video/mp4" />
+</video>
+
+For a quick comparison, let's check out, for instance, how the two different approaches behave on the most typical scenarios.
+
+<video style="margin-bottom: 24px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/nextgen-pickers/fields-common-usecases.mp4" type="video/mp4" />
+</video>
+
+Try out the live demo of the `DateField` below.
+
+<iframe
+  src="https://codesandbox.io/embed/date-field-demo-pb87v0?fontsize=12&hidenavigation=1&module=%2F"
+  style="width:100%; height:200px; border:0; border-radius: 4px; overflow:hidden;"
+  allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
+  sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+></iframe>
+
+### Fields roadmap
+
+The Fields are still in alpha, and there are many improvements on the way. The following list describes some of the most noticeable.
+
+- **Auto section switching**
+
+  So users don't need to manually navigate to the next section of the date after fulfilling the current one.
+
+- **Smart pasting**
+
+  Date strings pasted on Fields are parsed and matched with the sections of the date.
+
+- **Support days of the week**
+
+  Support selecting a date by navigating days of the week with arrow keys, or text editing.
+
+We'd love to see more of your [feedback](https://github.com/mui/mui-x/issues/new/choose) to help us improve these components even further.
+
+## Other Significant improvements
+
+Now that you've seen what's fundamentally different with the pickers, let us brief you on some other noteworthy refinements the components are getting.
+
+- **Customization of layout and internal components**
+
+  Combining the concept of slots and grid layout, you now can rearrange, extend, and customize most of the components used internally by the pickers views, and make them perfectly suited to your use cases.
+
+- **Default render input**
+
+  You don't need to provide the usual `TextField` on `renderInput` anymore unless you need it customized.
+  In which case you can do it by:
+
+  1. Customizing through props via `field` and `input` slots ([codebox example](https://codesandbox.io/s/customizing-fields-with-props-o66r1c?file=/demo.tsx)).
+  2. Build a new field component using the Field's headless API (documentation coming soon).
+  3. Use an entirely [custom text field](https://next.mui.com/x/react-date-pickers/date-picker/#custom-input-component) to suit your use cases.
+
+- **Drag to edit**
+
+  Editing a date range is even easier now with the new interface using drag and drop. Change `start` and `end` dates at will.
+
+<video style="margin-bottom: 24px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/nextgen-pickers/edit-drag.mp4" type="video/mp4" />
+</video>
```

Comment:
I think that the video is missing https://www.notion.so/mui-org/Blog-247ec2bff5fa46e799ef06a693c94917#7265c594d41e4906a521eb1a05caef53. At first, I thought that there was a bug with the selection state UI, but it's only because I didn't see when the right click was pressed. It feels clearer like this:

https://user-images.githubusercontent.com/3165635/210185968-b527d02b-a7af-451f-9e9a-aa3f3f0d4bf7.mov